scripts/start_mock_fxa_server.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
 
-. /app/venv/bin/activate
 python3 /app/tools/integration_tests/tokenserver/mock_fxa_server.py
```

Comment:
Optionally we could also kill this script since it was only needed for the venv? But I don't really care

Reply:
I'm tempted to leave it, partly because I can see it being useful in the future, at least before we drop the tokenserver.